user/tests.py

```diff
@@ -260,35 +260,49 @@ def setUp(self):
         )
         user.set_password('12345')
         user.save()
+
+        self.signin_url = reverse('user:signin')
     
     def test_get_signin_view(self):
-        response = self.client.get(reverse('user:signin'))
+        response = self.client.get(self.signin_url)
         self.assertEquals(response.status_code, 200)
     
     def test_valid_signin(self):
-        response = self.client.post(reverse('user:signin'), {'username':'test', 'password':'12345'})
+        response = self.client.post(self.signin_url, {'username': 'test', 'password': '12345'})
         self.assertEquals(response.status_code, 302)
         self.assertIn(SESSION_KEY, self.client.session)
         self.assertRedirects(response, reverse('user:home'))
     
     def test_nonexistent_user_signin(self):
-        response = self.client.post(reverse('user:signin'), {'username':'LeonardoDaVinci', 'password':'12345'})
+        response = self.client.post(self.signin_url, {'username': 'LeonardoDaVinci', 'password': '12345'})
         self.assertEquals(response.status_code, 200)
         self.assertNotIn(SESSION_KEY, self.client.session)
         self.assertFormError(response, 'form', '__all__', 'Please enter a correct username and password. Note that both fields may be case-sensitive.')
 
     def test_null_password_signin(self):
-        response = self.client.post(reverse('user:signin'), {'username':'test', 'password':''})
+        response = self.client.post(self.signin_url, {'username': 'test', 'password': ''})
         self.assertEquals(response.status_code, 200)
         self.assertNotIn(SESSION_KEY, self.client.session)
         self.assertFormError(response, 'form', 'password', 'This field is required.')
     
     def test_invalid_password_signin(self):
-        response = self.client.post(reverse('user:signin'), {'username':'test', 'password':'23456'})
+        response = self.client.post(self.signin_url, {'username': 'test', 'password': '23456'})
         self.assertEquals(response.status_code, 200)
         self.assertNotIn(SESSION_KEY, self.client.session)
         self.assertFormError(response, 'form', '__all__', 'Please enter a correct username and password. Note that both fields may be case-sensitive.')
-    
+
+
+class SignoutTests(TestCase):
+    def setUp(self):
+        user = CustomUser.objects.create(
+            username="test",
+            email="test@test.com",
+            phone= '',
+            date_of_birth= '1901-01-01'
```

Comment:
```suggestion
            phone='',
            date_of_birth='1901-01-01'
```
細かい点すみません！
SigninTestsとGetViewTests，EditViewTestsも同じズレがあるので修正お願いします！